lib/scrape.js

```diff
@@ -276,7 +276,7 @@ function checkGitHubUserCacheExpired(user) {
 }
 
 async function freshenUserGitHubCache(user, existingUser, organization) {
-  if (!existingUser) {
+  if (!existingUser || !existingUser.github_updated) {
```

Comment:
Minor change - I feel like this would be more elegant with `!(existingUser && existingUser.github_updated)`.

Reply:
Sure, works for me.